daemon/inertia/project/repository_test.go

```diff
@@ -82,7 +82,11 @@ func TestUpdateRepository(t *testing.T) {
 func TestCompareRemotes(t *testing.T) {
 	cwd, err := os.Getwd()
 	assert.Nil(t, err)
-	inertia, _ := path.Split(cwd)
+
+	// Traverse back down to base directory
+	inertia := path.Dir(cwd)
+	inertia = path.Dir(inertia)
+	inertia = path.Dir(inertia)
```

Comment:
Is there a better way to do this? It seems a bit fragile when we assume the base directory is exactly 3 levels deep.

Reply:
It certainly is, which is why it broke previously 😅 I think it's fine though - it's a test, and the directory structure of inertia in any environment (barring a structural change like this) should stay the same.

That said this was a very lazy fix, I'll change it to something a bit more clear:

```go
func TestCompareRemotes(t *testing.T) {
	// Traverse back down to root directory of repository
	repo, err := git.PlainOpen("../../../")
```

Good catch again, been getting sloppy 🙅‍♂️